notifications_utils/logging.py

```diff
@@ -40,43 +43,49 @@ def init_app(app, statsd_client=None, extra_filters: Sequence[logging.Filter] =
     app.config.setdefault("NOTIFY_APP_NAME", "none")
     app.config.setdefault("NOTIFY_LOG_PATH", "./log/application.log")
     app.config.setdefault("NOTIFY_RUNTIME_PLATFORM", None)
+    app.config.setdefault("NOTIFY_LOG_IGNORE_PATH_LIST", "/_status;/metrics")
```

Comment:
Suggest making this a list - I don't think we have anything that automatically reads flask config vars from env vars and if/where we added that we could do the list conversion.

This is the common pattern there seems to be for complex value parsing https://github.com/alphagov/notifications-api/blob/f5ff1aa6de14876cd8ed34b52caf552c1fa90142/app/config.py#L89

Reply:
Actually if you make it a `set` here you can avoid having to put together your dict down below

Reply:
Updated to a set.